src/components/Loans/OpenLoans/OpenLoansControl.js

```diff
@@ -227,15 +229,17 @@ class OpenLoansControl extends React.Component {
     });
   };
 
-  declareLost = loan => { this.props.declareLost(loan); };
+  declareLost = loan => this.props.declareLost(loan);
+
+  claimReturned = loan => this.props.claimReturned(loan);
 
   feefineDetails = (loan, e) => {
     const {
       resources,
       history,
       match: { params }
     } = this.props;
-    const accounts = get(resources, ['loanAccount', 'records'], []);
+    const accounts = resources?.loanAccount?.records || [];
```

Comment:
This looks so much cleaner 👏

Reply:
@SergiyVSergiyenko I wonder if we should use `nullish coalescing` in this case so:

 `resources?.loanAccount?.records ?? [];` 

I don't think it makes any difference in this example but perhaps keep it in consistent with other cases and having just one pattern will be better? What do you think?

Reply:
@mkuklis, thanks for the question. Since resources.loanAccount.records is an array (or undefined), and we do not expect to get an empty string or zero there, the `nullish coalescing` operator is not required. Although, in our case, it works as well.